src/pages/Recipes.js

```diff
@@ -70,7 +87,12 @@ export default function Recipes() {
           <span key={item}> {item}</span>
         ))}
       </StyledSection>
-      <button onClick={() => navigate("/")}>back</button>
+      <section>
+        {data.map((recipe) => {
+          return <RecipeCard recipeData={recipe} />;
```

Comment:
I think you forgot the `key` prop.
And you could use the implicit return and get rid of the return statement.